client/my-sites/people/people-list-item/index.jsx

```diff
@@ -42,13 +51,13 @@ export default React.createClass( {
 			this.userHasPromoteCapability() &&
 			! this.props.isSelectable
 		);
-	},
+	};
 
 	render() {
 		const canLinkToProfile = this.canLinkToProfile();
 		return (
 			<CompactCard
-				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove' ) }
+				{ ...omit( this.props, 'className', 'user', 'site', 'isSelectable', 'onRemove', 'translate', 'moment', 'numberFormat', 'recordGoogleEvent' ) }
```

Comment:
This is a huge list, are we sure we can't positively specify the props we want to hand down instead of negatively removing the ones we don't? `omit()` leaves us ignorant to what we are actually passing along.